### PR TITLE
Change the role "OWNER" to "ADMIN"

### PR DESCRIPTION
**Pull request by ryanpbrewster**
Sorry frontend/mobile

@cvializ @jaredpetker @DerekBurch @ajkochanowicz 

Merging this in 20 minutes or so.

### CODE REVIEW

**Comment by cvializ** [comment]
Will the actual owner still have an `Owner` role, or will we know they're the owner from the org's `ownerId`?

**Comment by ryanpbrewster** [comment]
The latter. They will have an `ADMIN` role.

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/OrganizationController.scala [hunk 1]
```diff
@@ -45,11 +45,13 @@ class OrganizationController @Inject() (
   }
 
   def modifyOrganization(pubId: PublicId[Organization]) = OrganizationUserAction(pubId, OrganizationPermission.EDIT_ORGANIZATION)(parse.tolerantJson) { request =>
+    println("[RPB] modifyOrg called with body: " + request.body)
```
Comment (maxov) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/OrganizationController.scala]: I don't think these should be there, they look like debug statements?

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/OrganizationController.scala]: Good catch, I was just going through to remove my debug statements.

**Pull request merged**